code/modules/modular_computers/file_system/programs/ntmessenger.dm

```diff
@@ -358,10 +358,18 @@
 	if (ringer_status)
 		computer.ring(ringtone)
 
+/// topic call that answers to people pressing "(Reply)" in chat
 /datum/computer_file/program/messenger/Topic(href, href_list)
```

Comment:
I will once again bring up, is this really worth having as a program?

This could be just be something directly onto tablets into tablets like PDA messaging was, I find it would make much more sense it being an app has no benefits.
This isn't like any other app, it's not supposed to take space, be deleted or transferred onto disks or borgs and whatnot, all it does is cause headaches because of snowflake code like this (This is the ONLY app that uses Topic(), by the way), when I find it would be easier for it to just be part of tablets.

Just as an example, in the past when you would sort all PDAs, you would check all PDAs and list them in the prefered order, using the individual PDA's owner.
Now you have to get all TabletMessengers, check each individual one's components for their drive, check if it exists, then check the drives if each one has the messenger app, then gets all the information off of the app to sort it.

I just think it being an app in general sucks, and fixing that would be a better change than this. While this PR does have a good fix, I think it will just demotivate the actual change I would like to see because sadly not many people want to improve things that already "work".

Reply:
wyci